MobiFlight/MidiBoard/MidiBoardManager.cs

```diff
@@ -21,48 +26,41 @@ public class MidiBoardManager
 
         public MidiBoardManager ()
         {
-            LoadDefinitions();
+            Load();
```

Comment:
Old name was easier to understand.

Reply:
Didn't you tell me to make them all Load() in a previous PR? We have four different classes that all load definition files. What do you want that method called so I can make it consistent in all four places?